### DISCUSSION TITLE
Add height to image

### DISCUSSION THREAD

**Discussion by rocramer** [spatie/laravel-medialibrary#2131] [Ideas] [answered]
The image tags do not contain a height attribute (only the width). This can lead to layout shifts while rendering. Google recommends adding the width and height attributes so modern browsers can calculate the aspect ratio automatically: https://web.dev/optimize-cls/#modern-best-practice

Is a PR welcome?

Thanks!

**Comment by freekmurze** [comment, marked as answer]
Send a PR to the v9 branch, and we'll review it there.
